examples/deploy_service/main.tf

```diff
@@ -38,18 +38,16 @@ provider "kubernetes" {
 data "google_client_config" "default" {}
 
 module "gke" {
-  source     = "../../"
-  project_id = "${var.project_id}"
-  name       = "${local.cluster_type}-cluster${var.cluster_name_suffix}"
-  region     = "${var.region}"
-  network    = "${var.network}"
-  subnetwork = "${var.subnetwork}"
-
-  kubernetes_version = "1.11.7-gke.12"
```

Comment:
Removed as it failed the build - this version is no longer supported for the given region.